app/src/main/res/values/dimens.xml

```diff
@@ -485,4 +485,10 @@
 
   <dimen name="home_fragment_padding_top">36dp</dimen>
   <dimen name="home_fragment_padding_bottom">148dp</dimen>
+
+  <dimen name="promoted_story_card_layout_marginEnd">8dp</dimen>
```

Comment:
Rename this and other values to 
`promoted_story_card_layout_margin_end`

Basically all dimens values should be written in following format:
1. all small characters
2. words separated by `_`

Reply:
ok sir

Reply:
For this you can make sure that all cards use a fixed width of `280dp` as we have seen in other 3 layouts.